styles/components/input.css

```diff
@@ -0,0 +1,91 @@
+.input-grp {
+  display: flex;
+  flex-direction: column;
+  width: 70%;
+}
+.input {
+  height: 40px;
+  width: 100%;
+  outline: none;
+  padding: 0 0.5rem;
+  margin: 0.4rem 0;
+  font-size: 0.9rem;
+  font-weight: 200;
+  border-radius: 0.5em;
+  border: 1px solid var(--color-gray-200);
+}
+.input:focus {
+  border: 1px solid var(--color-primary);
```

Comment:
It will be good, if you use all caps for naming a color variable. Like `--COLOR-PRIMARY`, as it increase readability that it is a variable.

Reply:
Sure i will do that

Reply:
I have fixed these variable naming can you please approve my PR